lib/pacemaker/pcmk_injections.c

```diff
@@ -120,7 +120,14 @@ pcmk__inject_failcount(pcmk__output_t *out, cib_t *cib_conn, xmlNode *cib_node,
             failcount = 0;
         }
     }
-    value = pcmk__itoa(failcount + 1);
+
+    if (pcmk_str_is_infinity(offset)) {
+        value = strdup(offset);
```

Comment:
We prefer `pcmk__str_copy()` now to assert on failure

Reply:
This is where the value of `offset` is technically used. But it's also fine to simply copy `PCMK_VALUE_INFINITY` based on the value of a `bool fatal` or `bool infinity` parameter.